docs/en/about-appium/getting-started.md

```diff
@@ -1,9 +1,13 @@
 ## Getting Started
 
-This doc will get you up and running with a simple Appium test and introduce
+This doc will get you up and running with a simple Appium 1.x test and introduce
 you to some basic Appium ideas. For a more comprehensive introduction to Appium concepts,
 please check out the [conceptual introduction](/docs/en/about-appium/intro.md).
 
+Note that this documentation explains about Appium 1.x.
```

Comment:
Notes could also be nicely formatted in markdown: https://stackoverflow.com/questions/50544499/how-to-make-a-styled-markdown-admonition-box-in-a-github-gist

Reply:
Yea, i like it. i haven't checked how it will be in https://appium.io/docs/en/about-appium/getting-started/ format tho.